chart/k8gb/Chart.yaml

```diff
@@ -46,6 +46,7 @@ annotations:
     - kind: DNSEndpoint
       name: dnsendpoints.externaldns.k8s.io
       version: v1alpha1
+      displayName: Gslb
```

Comment:
Should be DNSEndpoint

Reply:
thanks, I've actually fixed this before opening the PR, but forgot to "push-f" again :D